Content.Server/GameTicking/GameTicker.cs

```diff
@@ -635,11 +645,19 @@ public void EquipStartingGear(IEntity entity, StartingGearPrototype startingGear
             {
                 foreach (var slot in AllSlots)
                 {
-                    var equipmentStr = startingGear.GetGear(slot, profile);
+                    var equipmentStr = startingGear.GetGear(slot, profile, CaptainUniformPreference);
                     if (equipmentStr != "")
                     {
                         var equipmentEntity = _entityManager.SpawnEntity(equipmentStr, entity.Transform.Coordinates);
-                        inventory.Equip(slot, equipmentEntity.GetComponent<ItemComponent>());
+
+                        if (equipmentEntity.TryGetComponent(out ItemComponent? itemComponent))
+                        {
+                            inventory.Equip(slot, itemComponent);
+                        }
+                        else
+                        {
+                            Logger.Error("Equipment Entity " + equipmentEntity.Uid + " does not have ItemComponent, not equipping!");
```

Comment:
Logging UIDs is nigh useless.
```suggestion
                            Logger.Error($"Equipment Entity {equipmentEntity} does not have ItemComponent, not equipping!");
```